MIGRATION.md

```diff
@@ -0,0 +1,65 @@
+  - The new dynamic configuration system allows us to publish the dev version of the orb _before_ calling it for testing. This means that the 90 day limit is no longer an issue.
+- **Adopting Tag/Release based publishing**
+  - Publishing previously required a special text flag to be added to the commit message and a new version was published on every merge to the main branch.
+  - The new tag/release based publishing system will simply publish your orb when you opt to push a versioned tag or use GitHub's [releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) feature, which will create a tag and give you an opportunity to create a change log via [release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes).
+- **Review system**
+  - The new "review" job can automatically detect opportunities to improve best-practices and provide native JUNIT output which will be displayed in the CircleCI UI.
+  - Modeled after _shellcheck_, it is easy to skip any "review check" by supplying its "RC" code in the "exclude" parameter of the job.
+- **Simplified/Improved PR Commenting**
+  - Automatically comment on the PR associated with a commit when each new orb version is published (dev or production.)
+  - The comment will include a link to the Orb Registry to preview dev versions of the orb, and a live link to the production version of the orb.
+
+## How to Migrate
+
+1. Enable "[dynamic configuration](https://circleci.com/docs/2.0/dynamic-config/#getting-started-with-dynamic-config-in-circleci)" for your project on CircleCI.
+   - Visit https://app.circleci.com/ and navigate to your project.
+   - Click _Project Settings_ in the upper right corner.
+   - Click _Advanced_
+   - Toggle on _Enable Dynamic Configuration_
+2. Clone your orb project locally.
+3. Create a new branch for upgrading your pipeline
+   - `git checkout -b orb-tools-11-migration`
+4. Copy the `migrate.sh` script from this repository into your project's root directory.
+5. Run the script
+   - `chmod +x migrate.sh`
+   - `bash migrate.sh`
+6. After executing the script:
+
+   - The script will ask you for some basic information about your orb, such as the namespace, name of the orb, and name of your publishing context. All of this information is present in your existing configuration.
+   - Your existing configuration will be renamed to `config.yml.bak`
+   - Two new configuration files will be downloaded from the template repository and modified with your inputs.
+   - The migrate script will self-delete.
+
+7. _(OPTIONAL)_ Enable PR comments.
+
+   - If a GitHub token is provided, when a new dev or production version of your orb is published, a comment will be added to the PR associated with the commit.
```

Comment:
Consider making this more explicit:

When a new dev or production version of your orb is published, a comment will be added to the PR associated with the commit. **NOTE:** _This will only be enabled if a GitHub token is provided_